srcpkgs/llvm17/template

```diff
@@ -599,7 +623,7 @@ libcxxabi_package() {
 
 libcxxabi-devel_package() {
 	short_desc+=" - low level support for libc++ - development files"
-	depends="libcxxabi>=${version}_${revision}"
+	depends="libcxxabi-${version}_${revision} llvm17-${version}_${revision}"
```

Comment:
I don't think `llvm-libunwind-devel`, `libcxxabi-devel`, `libcxx-devel` should depend on llvm17.